docs/json_rpc_api.mediawiki

```diff
@@ -450,14 +450,14 @@ the method name for further details such as parameter and return information.
 # <code>inputs</code>: <code>(json array, required)</code> The inputs to the transaction of type.
 #: <code>txid</code>: <code>(string)</code> Unspent tx output hash.
 #: <code>vout</code>: <code><numeric></code> Index of the output being redeemed.
-#: <code>amt</code>: <code><numeric></code> Amount of utxo.
+#: <code>amt</code>: <code><numeric></code> Amount of utxo in atoms.
 #: <code>tree</code>: <code><numeric></code> Which tree utxo is located.
-# <code>amount</code>: <code>(json object, required)</code> json object with the destination addresses as keys and amounts as values.
-#: <code>{"desaddr":"amount", ...}</code>
+# <code>amount</code>: <code>(json object, required)</code> json object with the destination address as key and amount as value in atoms.
+#: <code>{"desaddr":"amount"}</code>
```

Comment:
I realize it already said `desaddr`, but what do you think about making it match the CLI help output of `address` or `addr`?

```bash
$ dcrctl help createrawsstx
createrawsstx [{"txid":"value","vout":n,"tree":n,"amt":n},...] amount [{"addr":"value","commitamt":n,"changeaddr":"value","changeamt":n},...]

...
Arguments:
...
2. amount (object, required) JSON object with the destination addresses as keys and amounts as values
{
 "address": n.nnn, (object) The destination address as the key and the amount in DCR as the value
 ...
}
...

Result:
"value" (string) Hex-encoded bytes of the serialized transaction
```

Reply:
Also added the map breakdown to the first line after calling `help createrawsstx`